### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by tddschn**
Fix typos in the source code.

### CODE REVIEW

File: easygraph/functions/graph_embedding/sdne.py [hunk 1]
```diff
@@ -18,18 +18,18 @@ def get_relation_of_index_and_node(graph):
     return idx2node, node2idx
 
 
-def l_2nd(beta):
+def l_2and(beta):
     from tensorflow.python.keras import backend as K
 
-    def loss_2nd(y_true, y_pred):
```
Comment (coreturn) [inline easygraph/functions/graph_embedding/sdne.py:24]: I think the "2nd" means second?

**Comment by coreturn** [comment]
Thanks for your commit, the code has been reviewed, and just one small detail need to be checked. maybe you can modify it and commit again. Btw, what tool do you use to check the typo, i think it can be useful.

File: easygraph/functions/graph_embedding/sdne.py (hunk 1 above)
Comment (tddschn) [reply easygraph/functions/graph_embedding/sdne.py:24]: Yes you're right. The typos fixed are made automatically and I didn't review them all...

I'll double check the changes. :)

**Comment by tddschn** [comment]
Real-time typo checking: cSpell VS Code extension;
Batch auto fixing: `typo-cli`, you can get it via `cargo install typos-cli`.

**Pull request merged**